docs/src/main/asciidoc/deploying-to-openshift-native-howto.adoc

```diff
@@ -0,0 +1,118 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="deploying-to-openshift-native-howto"]
+= Deploy {project-name} applications compiled to native executables
+
+include::_attributes.adoc[]
+:diataxis-type: howto
+:categories: cloud, native
+:summary: This guide describes how to deploy a Quarkus application to {openshift} compiled to native executables.
+:topics: devops,kubernetes,openshift,cloud,deployment
+:extensions: io.quarkus:quarkus-openshift
+
+You can deploy your native {project-name} applications to {openshift} compiled to native executables by using the Docker build strategy.
+
+You must create a native executable for your application that targets the Linux AMD64 operating system.
+If your host operating system is different from this, create a native Linux executable by using a container runtime, for example, Docker or Podman.
+
+Your Quarkus project includes pregenerated Dockerfiles with instructions.
+If you want to use a custom Dockerfile, add the file to the `src/main/docker` directory or any location inside the module.
+Additionally, set the path to your Dockerfile by using the `quarkus.openshift.native-dockerfile` property.
```

Comment:
If I want to use custom Dockerfile, I can simply replace the original one and  don't sent any property. Maybe you should advise to set `quarkus.openshift.native-dockerfile` if you want to have multiple docker files and switch between them using this configuration property? I think it is confusing to keep the original file if it is not used as when someone else come to my project, they won't realize that is the dead file.

Reply:
Updated per your feedback.